db/migrate/20210322075024_add_image_url_to_rulebooks.rb

```diff
@@ -0,0 +1,5 @@
+class AddImageUrlToRulebooks < ActiveRecord::Migration[6.1]
```

Comment:
please merge the migrations into one
delete this file and add the content to the previous migration
you don't need to migrate again as your table will be correctly setup, but for others who will migrate after pulling master it's one less file to migrate (and therefore faster)